.changeset/old-coats-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+Add authorization header on techdocs api requests
```

Comment:
Perhaps mention that the client now needs the identity API, and that's why it's a breaking change.

Reply:
As the identity api provisioning is done in plugins/techdocs/src/plugin.ts, do you want me to change it to a patch change? It is a backwards compatible change otherwise? techdocs-backend is missing to, need to add that.

Reply:
Strictly speaking it's breaking because the APIs and ApiRefs are exported from techdocs, meaning you could be calling e.g. `new TechDocsApi` from the app to instantiate a custom version of it. It's pretty unlikely that that's being done atm because both APIs only depend on other APIs, leaving little reason to instantiate them manually. But nonetheless they're exported, so it's a breaking change ¯\\\_(ツ)\_/¯